tools/sideloading/install.py

```diff
@@ -0,0 +1,53 @@
+import sys

```

Comment:
You can add shebang `#!/usr/bin/env python3` to make script runnable 
If you want to use sideloading with multiple applications, you could then create a link to `install.py` in your local bin path and run it from anywhere.